action.yml

```diff
@@ -7,26 +7,50 @@ inputs:
   gitleaks_license:
     required: true
     description: Gitleaks license key to use the licensed version.
-  gitleaks_regex_internal_url:
+  gitleaks_config_contents:
     required: false
     default: ""
-    description: Regex to identify internal urls
+    description: Full contents of a gitleaks config file (multiline)
+  gitleaks_config_vars:
+    required: false
+    default: ""
+    description: JSON object containing key/value pairs for template substitution
+  gitleaks_extend_config:
```

Comment:
Not sure why this has been controlled through input parameter, do we have any usecase where anyone want to run scan without built in rules. 

If we really want to give customisation to control this behaviour, I feel default should be true